Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:20-slim
+
+EXPOSE 8600
+
+WORKDIR /root/retro-env-can-weather-chan
+RUN apt update && apt install git tini -y
```

Comment:
Does the `node:20-slim` image come with Yarn by default?

Reply:
It sure does! Mentioned here: https://github.com/nodejs/docker-node/blob/main/README.md#image-variants